system/lib/libc/musl/src/fcntl/fcntl.c

```diff
@@ -8,10 +8,14 @@
 int fcntl(int fd, int cmd, ...)
 {
 	unsigned long arg;
-	va_list ap;
-	va_start(ap, cmd);
-	arg = va_arg(ap, unsigned long);
-	va_end(ap);
+	// XXX Emscripten: According to the va_arg man page it is undefined behaviour to
+	// read arguments that are not passed.
```

Comment:
```suggestion
	// read arguments that are not passed in. This can lead to a false positive
	// in SAFE_HEAP, so avoid it.
```

Reply:
Done.  Although I'm not sure it really a false positive.. reading and arg when none exists is just wrong/undefined.